README.md

```diff
@@ -100,18 +68,64 @@ Much of the philosophy behind Bedrock is inspired by the [Twelve-Factor App](htt
+
+You can learn more about WordPress configuration with Bedrock in the [Bedrock Configuration docs](https://docs.roots.io/bedrock/master/configuration/).
+
+### Understanding the WordPress codebase
+
+Bedrock installs WordPress as a required package so updates can be managed by Composer. As such, the contents of the `wp-content` directory have been moved outside the WordPress codebase so changes can be made safely to files within those directories without conflicts.
+
+* Theme are installed into `web/app/themes/`.
+* Plugins are installed into `web/app/plugins`.
+* Must-use plugins are installed into `web/app/mu-plugins`.
+* The WordPress admin dashboard is available at `https://example.com/wp/wp-admin/`
+
+### Using Composer to manage plugins and themes
+
+[Packagist](https://packagist.org) is a repository of Composer packages that are available by default to projects managed by Composer. Packagist libraries receive updates from their source GitHub repositories automatically.
+
+[WPackagist](https://wpackagist.org) is a Packagist-like mirror of the WordPress.org [plugin](https://wordpress.org/plugins) and [theme](https://wordpress.org/themes) repositories and is included with Bedrock out of the box. 
+
+You may install packages from Packagist or WPackagist without any additional configuration using `composer require`.
+
+#### Requiring a package from Packagist
+
+Some WordPress developers push their packages to Packagist in addition to the WordPress plugin and theme repositories. In this way, it may be beneficial to pull those packages directly from Packagist to get the latest code directly from the source.
+
+```
+composer require yoast/wordpress-seo
+```
+
+Packages that are flagged as `wordpress-plugin`, `wordpress-theme` or `wordpress-muplugin` in their `composer.json` files will be installed automatically in the appropriate `web/app/` directory by Composer.
+
+#### Requiring a package from WPackagist
+
+For all other plugins and themes that are not managed on Packagist, you can use `composer require` as well, using `wpackagist-plugin` or `wpackagist-theme` as the vendor and the plugin or theme slug as the package name.
+
+```
+composer require wpackagist-theme/twentytwentytwo
+```
+
+```
+composer require wpackagist-plugin/advanced-custom-fields
+```
+
+## Contributing
+
+Contributions are welcom in the form of GitHub pull requests. However, the `pantheon-upstreams/wordpress-composer-managed` repository is a mirror that does not directly accept pull requests.
 
-3. Add theme(s) in `web/app/themes/` as you would for a normal WordPress site
-4. Set the document root on your webserver to Bedrock's `web` folder: `/path/to/site/web/`
-5. Access WordPress admin at `https://example.com/wp/wp-admin/`
+Instead, to propose a change, please fork [pantheon-systems/wordpress-composer-managed](https://github.com/pantheon-systems/wordpress-composer-managed) and submit a PR to that repository.
```

Comment:
If I wasn't a Pantheor and wanted to make a contribution, should I fork [pantheon-systems/wordpress-composer-managed](https://github.com/pantheon-systems/wordpress-composer-managed) to my own account and then submit the PR **to** [CdrMarks/wordpress-composer-managed](https://github.com/CdrMarks/wordpress-composer-managed)?

Reply:
This was copied from the Drupal Composer Managed readme, but my understanding is that you'd fork, commit changes to your fork (or branch) and then submit a PR off of your fork

Reply:
After reading this and others examples of this language, I'm going to leave this part alone and resolve this comment.